packages/launch/src/focused-launch/summary/index.tsx

```diff
@@ -518,12 +520,12 @@ const Summary: React.FunctionComponent = () => {
 	return (
 		<div className="focused-launch-summary__container">
 			<div className="focused-launch-summary__section">
-				<Title>
+				<Title tagName="h2">
 					{ hasPaidDomain && hasPaidPlan
 						? __( "You're ready to launch", __i18n_text_domain__ )
 						: __( "You're almost there", __i18n_text_domain__ ) }
 				</Title>
-				<p className="focused-launch-summary__caption">
+				<SubTitle tagName="p" className="focused-launch-summary__caption">
```

Comment:
Could also remove the `font-size` and `color` CSS rules from the `.focused-launch-summary__caption` ? We don't need them anymore, now that we're using `<Subtitle />`

Reply:
Makes total sense and I removed them!